Dockerfile

```diff
@@ -136,38 +134,7 @@ RUN ./autogen.sh && \
     make && \
     make DESTDIR=/output install-strip
 WORKDIR /protobuf/python/
-# Protobuf is using a deprecated technique to install Python package with
-# easy install. This is causing issues since 2021-04-21. (https://discuss.python.org/t/pypi-org-recently-changed/8433)
-# We have to install pip and install six ourselves so we do not trigger the
-# broken protobuf install process.
-RUN pip3 install --user --ignore-installed wheel six && \
-    python3 setup.py install --user --cpp_implementation
-# We'll finish up the process of building protobuf below, but first, a bit of
-# explanation.
-#
-# Since we can't use `pip` with protobuf's `setup.py`, the package gets
-# installed via setuptools in an "egg" - a directory which acts as a python
-# module, but isn't a package and thus isn't automatically part of the package
-# namespace. Eggs need their contents to be added to python's `sys.path` to
-# become visible. Hooks that run early during python startup normally read
-# `setuptools.pth` and other `.pth` files and add all of the directories
-# referenced in those files to the path; egg contents are placed in those files
-# to be made available to the rest of the python installation.
-#
-# In multistage builds this doesn't work so well, because those `.pth` files can
-# easily be overwritten by other versions from a different image, and the eggs
-# we're installing here will no longer appear in `sys.path`. To work around
-# that, we just copy the contents of the existing `.pth` files into a new one
-# with a name we're sure won't be overwritten. Python will read this new `.pth`
-# file at startup and include the eggs in the path.
-#
-# If you're wondering why the `grep -v` is necessary, meditate upon the fact
-# that these files aren't merely metadata but also executable code, and
-# setuptools, by design, uses this feature to inject code into every python
-# program that runs on your system.
-RUN export PYTHON3_VERSION=`python3 -c 'import sys; version=sys.version_info[:3]; print("python{0}.{1}".format(*version))'` && \
-    cd /output/usr/local/lib/$PYTHON3_VERSION/site-packages&& \
-    cat *.pth | grep -v "import sys" | sort -u > docker_protobuf.pth
+RUN pip3 install --user protobuf==3.18.1
```

Comment:
I think the intent here was to install the package from source using the checked out repo (hence why the workdir is set to `/protobuf/python/`). Not sure how important that is, or whether it can be done without a complex workaround.